rolling/distribution.yaml

```diff
@@ -2773,6 +2773,21 @@ repositories:
       url: https://github.com/DFKI-NI/mir_robot.git
       version: rolling
     status: developed
+  mobile_catkin_modules_build_development_tools:
+    doc:
+      type: git
+      url: https://github.com/Application-UI-UX/mobile_catkin_modules_build_development_tools.git
+      version: main
+    release:
```

Comment:
At this stage for submitting to rolling please submit it without the release stanza. Once that is merged, then you can ask for the ros2-gbp to becreated for you. And then you can make a binary release with bloom.

Reply:
@tfoote you said that it does not have to be ros2-gbp and can be can it be another repository under my control; little confused about that part.